### PR TITLE
UGENE-7532: UGENE can set font point size to 0 in viewers and make th…

### PR DESCRIPTION
**Pull request by yalgaer**
…em inaccessible

### CODE REVIEW

File: src/corelibs/U2View/src/ov_msa/MaEditor.cpp [hunk 1]
```diff
@@ -463,7 +463,7 @@ void MaEditor::updateFontMetrics() {
     const int minimumSafeFontPointSize = 8;  // This value was historically used in UGENE as minimum with no known issues.
     QFont fontToEstimate = font;
     int estimatedMinimumFontPointSize = minimumSafeFontPointSize;  // Start with a safe value and estimate smaller values.
-    while (fontToEstimate.pointSize() > 0) {
+    while (fontToEstimate.pointSize() > 1) {
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/MaEditor.cpp:466]: The error sign is `QFont::setPointSize: Point size <= 0 (0), must be greater than 0`.
The point size here is already greater than 0, you made them greater than 1. I don't see how it helps.

**Review by DmitriiSukhomlinov — CHANGES_REQUESTED** [review]
Some strange fix, without any proves that it really helps, disapproved.

**Comment by DmitriiSukhomlinov** [comment]
Also, don't forget, that the original problem has appeared on Windows, so it should be reproducible on this system.

File: src/corelibs/U2View/src/ov_msa/MaEditor.cpp (hunk 1 above)
Comment (yalgaer) [reply src/corelibs/U2View/src/ov_msa/MaEditor.cpp:466]: Could you please check the algorithm beyond the changed line? The only reason to check the value for 1 here is to decrease it by -1 3 lines later.

**Comment by yalgaer** [comment]
> Also, don't forget, that the original problem has appeared on Windows, so it should be reproducible on this system.

It is reproducible on any OS, because it depends on font metrics only. Have you tried the font I mentioned?

**Pull request merged**